storybookjs/storybook
# Controls: Fix color control not resetting when initial value is defined

https://github.com/storybookjs/storybook/issues/17171

## What I did

Update useEffect hook in Color.tsx file to set the value of color input to initial value

## How to test

- Run `yarn storybook:ui` in `code` dir
- Navigate to `http://localhost:6006/?path=/story/addons-controls-basics--undefined`
- Verify that resetting works on color control
- DEMO: https://www.loom.com/share/b71ddb749f9a4b5fb1e3081396c727bb
